src/Automatonymous/Builders/InternalStateMachineEventActivitiesBuilder.cs

```diff
@@ -0,0 +1,174 @@
+﻿using Automatonymous.Binders;
+using System;
+using System.Collections.Generic;
+using System.Linq.Expressions;
+
+namespace Automatonymous
+{
+    internal class InternalStateMachineEventActivitiesBuilder<TInstance> : StateMachineEventActivitiesBuilder<TInstance>
```

Comment:
They're internal, sure, but I'd like to see all of the builder interfaces in a namespace, like Configuration/Builder or something, to avoid cluttering the root namespace.